### ISSUE TITLE
PostgreSQL 12 DDL Issue (new "GENERATED"  statement)

### ISSUE THREAD

**Issue by ncister** [dbeaver/dbeaver#7861]
Hi
I'm using a **PostgreSQL 12.1** Instance
If i execute the follow DDL command:
```
CREATE TABLE ts_test (
	f1 int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
	f2 int4 NULL,
	ts timestamptz GENERATED ALWAYS AS (fn_current_timestamp()) STORED,
	CONSTRAINT pk_ts_test PRIMARY KEY (f1)
);
```
... when I use "GENERATE SQL" I obtain:

```
CREATE TABLE ts_test (
	f1 int4 NOT NULL GENERATED ALWAYS AS IDENTITY,
	f2 int4 NULL,
	ts timestamptz NULL DEFAULT fn_current_timestamp(),
	CONSTRAINT pk_ts_test PRIMARY KEY (f1)
);
```
The problem is with the statement: **GENERATED ALWAYS AS (fn_current_timestamp()) STORED**

Again: using "GENERATED ALWAYS AS IDENTITY"  (instead of "SERIAL") the sequence objects are managed "internally" .... so, other tools (for example pgadmin4), don't list them in the "Sequences" tab.

Regards.

**Comment by LonwoLonwo** [comment]
https://www.postgresql.org/docs/current/ddl-generated-columns.html table for tests can be taken from here.

**Comment by LonwoLonwo** [comment]
Fixed in 21.0.1

**Comment by uslss** [comment]
verified
